cmd/context/utils.go

```diff
@@ -196,6 +196,9 @@ func GetManifest(ctx context.Context, srcFolder string, opts ManifestOptions) (*
 			log.Infof("could not load manifest: %s", err.Error())
 		}
 
+		if manifest.Name == "" {
```

Comment:
I think we shouldn't have `name` in the manifest yet

Reply:
We can have it if the manifest is the old one where the name was mandatory

Reply:
But that should be part of `dev`, no?
I think we should remove this one:
https://github.com/okteto/okteto/blob/master/pkg/model/manifest.go#L20